### ISSUE TITLE
Question about implementation of log var network

### ISSUE THREAD

**Issue by ZhuFengdaaa** [wyndwarrior/Sectar#3]
Why log var network is defined as a single parameter(its expression ability seems being limited compared with a neural network) and initialized with log(0.1) ?  https://github.com/wyndwarrior/Sectar/blob/2e0d67481a/exps/swimmer/swimmer_explore.py#L92

**Comment by jcoreyes** [comment]
We found the single parameter version to work better than the neural network version which is more expressive but we found more difficult to optimize in our experiments. Initializing it to log(0.1) affects the initial learning rate (smaller variance will create a larger loss) and we found that value to work well.

**Comment by ZhuFengdaaa** [comment]
Thanks.
